### ISSUE TITLE
pruning  is useless

### ISSUE THREAD

**Issue by eeric** [Syencil/mobile-yolov5-pruning-distillation#5]
prunable_modules = []
        prunable_module_type = (nn.BatchNorm2d, )
        for i, m in enumerate(model.modules()):
            if i in ignore_idx:
                continue
            if isinstance(m, prunable_module_type):
                prunable_modules.append(m)

as mentioned above, model structure was not change! 
in addtion, BN of conv module was not appended prunable_modules.
where pruning?

**Comment by Syencil** [comment]
u may run fuseBN before pruning. plz check ur model definition. i think there is no problem in codes. u can also try mobilev2-yolo5s. :)

**Comment by eeric** [comment]
sorry, I only used sparse learning not using prune.
